wordpress-mobile/WordPress-Android
# Jetpack Kotlinization: StatsConnectJetpackActivity

Parent #14904

This PR converts the Java `StatsConnectJetpackActivity.java` file to the Kotlin `StatsConnectJetpackActivity.kt` file.

To test:
- Launch the app.
- Login with a site that doesn't have Jetpack installed or activated (I used a `Pressable` site). PS: You can also deactivate the Jetpack plugin (if need be).
- Navigate to `My Site` -> `Stats`.
- The `INSTALL JETPACK` empty screen should be shown (see screenshot below).
- Interact with this screen and verify that everything works as expected.
  - Clicking or swapping back should navigate you back.
  - Clicking the `INSTALL JETPACK`  should navigate you to the `Install Jetpack` screen.
  - Clicking the `Terms and conditions` link should open the `Terms of Service` page on your browser.
  - Clicking the `JETPACK FAQ` link should open the `Jetpack FAQ` page on your browser.

The `INSTALL JETPACK` screen:

<img src="https://user-images.githubusercontent.com/9729923/123387906-f2356000-d5a0-11eb-975d-dc45743fe4d9.png" width="250" height="500">

## Regression Notes
1. Potential unintended areas of impact

N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Manually tested as described above in the `To test` section.

3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
